packages/sitecore-jss/src/layout/models.ts

```diff
@@ -4,6 +4,7 @@
 export interface LayoutServiceData {
   sitecore: LayoutServiceContextData & {
     route: RouteData | null;
+    tracked?: boolean;
```

Comment:
Here are the possible cases:
- `GraphQLLayoutService` service: never tracks
- `RestLayoutService`: does not track if `tracking: false` is passed to the constructor (otherwise tracks)

In `page-props-factory.ts`  I do not know whether `GraphQLLayoutService` or `RestLayoutService` is configured.

I have re-implemented this part. Instead of response modification, I have added the `tracking` field to `LayoutService` interface.